www/templates/html/DefaultHomepage.tpl.php

```diff
@@ -39,8 +39,8 @@
     <div class="mh-featured">
       <a class="dcf-d-flex dcf-flex-col dcf-ai-center dcf-txt-decor-hover" href="<?php echo $baseUrl ?>search/">
         <div class="mh-featured-icon mh-green dcf-d-flex dcf-ai-center dcf-jc-center dcf-h-10 dcf-w-10 dcf-circle">
-          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>/templates/html/css/images/play-icon.svg"  title="Play Icon">
-            <img src="<?php echo $baseUrl; ?>/templates/html/css/images/play-icon-white.png" alt="browse media">
+          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>templates/html/css/images/play-icon.svg"  title="Play Icon">
+            <img src="<?php echo $baseUrl; ?>templates/html/css/images/play-icon-white.png" alt="browse media">
```

Comment:
The SVG icons appear as white squares now. I assume it has to do with the updated file paths?